src/components/Explore/Axes.tsx

```diff
@@ -7,31 +7,29 @@ import {
   getXTickFormat,
   getTimeAxisTicks,
 } from 'components/Charts/utils';
-
-const yTickFormatDefault = (value: number) => value.toString();
+import { formatPercent } from 'common/utils';
+import { DataMeasure } from './interfaces';
 
 const Axes: React.FC<{
   height: number;
   dateScale: ScaleTime<number, number>;
   yScale: ScaleLinear<number, number>;
   isMobile: boolean;
   yNumTicks?: number;
-  yTickFormat?: (value: number) => string;
-}> = ({
-  height,
-  dateScale,
-  yScale,
-  isMobile,
-  yNumTicks = 10,
-  yTickFormat = yTickFormatDefault,
-}) => {
+  dataMeasure: DataMeasure;
```

Comment:
How about passing the formatting function directly instead? That way this component is a bit more generic and can support other specialized formatters if we need to

```tsx
const Axes: React.FC<{
  /* ... */
  yTickFormat: (value: number) => string;
}>
```